src/mechanics/mesh/MeshFem.cpp

```diff
@@ -99,7 +99,18 @@ Group<NodeSimple> MeshFem::NodesTotal(DofType d)
 {
     Group<NodeSimple> group;
     for (auto& element : this->Elements)
-        for (int iNode = 0; iNode < element.DofElement(d).Interpolation().GetNumNodes(); ++iNode)
+    {
+        const ElementFem* dofElm;
+        try
+        {
```

Comment:
I did not want to enclose the whole loop in a try catch block
because I am only looking for the out of range exception thrown 
by the DofElement getter.